two-entity-poc/src/models/blog.ts

```diff
@@ -0,0 +1,54 @@
+import { CreationOptional, DataTypes, InferAttributes, Model } from "sequelize";
+
+import {sequelize} from "../util/database";
+import IBlog from "../interfaces/blog-interface";
+import { EMPTY_STRING_REGEX } from "../constants/regex";
+import { ERROR_MESSAGES } from "../constants/validation-messages";
+class Blog extends Model<InferAttributes<IBlog>> {
+  declare id: CreationOptional<number>;
+  declare title: String;
```

Comment:
I think we saw an error, which suggested to use string instead of String.
so you can update this code.

Reply:
Updated.